internal/pkg/service/common/httpserver/middleware/otel.go

```diff
@@ -53,7 +62,14 @@ func OpenTelemetry(tp trace.TracerProvider, mp metric.MeterProvider, opts ...OTE
 			ctx = context.WithValue(ctx, redactedRouteParamsCtxKey, cfg.redactedRouteParams)
 
 			// Process request
-			next.ServeHTTP(w, req.WithContext(ctx))
+			startTime := time.Now()
+			rw := negroni.NewResponseWriter(w)
```

Comment:
zajímavé že to musíš ta wrapnůt, ale negroni je první odpověď na stack overflow takže cajk :)

Reply:
jj, ono ten middleware ide vlastne iba jednym smerom, nie je nic take ako `response`, ... je iba `response writer` (`w`) a ak chces vediet status code, tak ho musis wrapnut. Myslim, ze hlavny dovod je streaming, ... ono status kodov moze byt aj viac, ... napr `103 (Early Hints)`.